modules/mobile/app/serializers/mobile/v0/dependents_request_decisions_serializer.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class DependentsRequestDecisionsSerializer
+      include JSONAPI::Serializer
+
+      set_id :id
+
+      set_type :dependents_request_decisions
+
+      attribute :dependency_verifications
+      attribute :diaries
+      attribute :prompt_renewal
+
+      def initialize(user_uuid, drd_info)
+        resource = DependentsRequestDecisionsStruct.new(
+          id: user_uuid,
+          dependency_verifications: Array.wrap(drd_info[:dependency_decs]).map do |a|
```

Comment:
if there's only one of these, then it doesn't come in array. Mirroring other controllers logic that uses this service to remove social security number.